Solutions/OCILogs/Parsers/OCILogs.txt

```diff
@@ -0,0 +1,21 @@
+// Usage Instruction :
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as OCILogs.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. OCILogs | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+OCI_Logs_CL
```

Comment:
This is not an ASIM parser, and the review is not intended to make it compliant with ASIM.